mitmproxy/builtins/script.py

```diff
@@ -16,6 +16,19 @@
 from watchdog.observers import polling
 
 
+class NS:
```

Comment:
It lets us access the event objects in the namespace dictionary as attributes. This unifies the interface with Addons, which means that ns can just be a class if it's been replaced by the script .start.